src/sheet/SheetEditor.js

```diff
@@ -350,8 +350,25 @@ export default class SheetEditor extends AbstractEditor {
     let editorSession = this.getEditorSession()
     let cell = this._getAnchorCell()
     let newValue = this._formulaEditorContext.node.getText()
+    let { anchorRow, anchorCol} = this._getSheetSelection()
+
+    // Select cell below
+    anchorRow += 1
+
     editorSession.transaction(tx => {
       tx.set(cell.getPath(), newValue)
+      tx.setSelection({
```

Comment:
There is a method in SheetComponent, which also shifts the viewport:
```
this.refs.sheet._nav(1, 0)
```
On the long run we should tidy-up the API of SheetComponent

Reply:
Wanted to do it one flow. I guess we should have the _nav as a utility returning a new selection, and handle the this.send stuff in a separate method.

Reply:
I see. Yeah, I need to think how to make the SheetComponent API useful.
So let's stick with your implementation, and fix SheetComponent in a different iteration